pkg/config/runtime_config.go

```diff
@@ -19,5 +19,5 @@
 package config
 
 type RuntimeConfig struct {
-	Checks map[string]any `yaml:"checks"`
+	Checks map[string]any `yaml:"checks" json:"checks"`
```

Comment:
Added the json tag to be able to test using the convenience method httpmock.NewJsonResponder. Else we would have to write a YAML text (which is a hassle to update each time) and use a String Responder each time. I'm open to other suggestions :)

Reply:
It's okay imo, but as a suggestion to split concerns you could write your test input, marshal it to yaml and then give the marshaled yaml to the string responder.
```go
		out, err := yaml.Marshal(tt.response)
		resp := httpmock.NewStringResponder(tt.code, string(out))
```

Reply:
Sure, that sounds more than reasonable 😅

Reply:
I used the ByteResponder, to avoid casting to string.